### ISSUE TITLE
FEAT | Cinematic module

### ISSUE THREAD

**Issue by louisgv** [nujas/TheJanusFramework#2]
This would either belong in a cinematic module or in the animation module... Think it might be more beneficial to split

**Comment by DennisSSDev** [comment]
I'm assuming you're talking about the find target with axis. It doesn't make sense to be inside the cinematic module, this feature is deeply connected with player input and gameplay. it should remain in the Dynamic Targeting Module

**Comment by louisgv** [comment]
No, I'm referring to the cinematic binding for QTE and switch camera stuff
